src/_webrsp.m

```diff
@@ -273,10 +297,12 @@ N ARGS S ARGS("*")=$E(HTTPREQ("path"),2,9999)
  E  D W("Content-Type: application/json; charset=utf-8"_$C(13,10))
  ;
  ; Add CORS Header
- I $G(HTTPREQ("method"))="OPTIONS" D W("Access-Control-Allow-Methods: OPTIONS, POST"_$C(13,10))
- I $G(HTTPREQ("method"))="OPTIONS" D W("Access-Control-Allow-Headers: Content-Type"_$C(13,10))
- I $G(HTTPREQ("method"))="OPTIONS" D W("Access-Control-Max-Age: 86400"_$C(13,10))
- D W("Access-Control-Allow-Origin: *"_$C(13,10))
+ I ^%webhttp(0,"cors","enabled")="Y" D
+ . I $G(HTTPREQ("method"))="OPTIONS" D W("Access-Control-Allow-Credentials: "_^%webhttp(0,"cors","credentials")_$C(13,10))
```

Comment:
This needs to be figured out how to make this work with out globals. I have a specific application that needs to be able to do CORS and work without globals, the previous code worked - though incomplete.

Reply:
Done